connector/sumconnector/metadata.yaml

```
@@ -0,0 +1,13 @@
+type: sum
+scope_name: otelcol/sumconnector
+
+status:
+  class: connector
+  stability:
+    development: [traces_to_metrics, metrics_to_metrics, logs_to_metrics]
+  distributions: [contrib]
```

```suggestion
  distributions: []
```
We don't want to advertise its inclusion in `contrib` until it's included in `contrib` releases ([here](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml)), otherwise users can get pretty confused by this.